packages/former/src/components/FormContent.vue

```diff
@@ -31,6 +47,15 @@ function getFormKitId(node: any): string | undefined {
   return isFormKitSchemaNode(node) ? (node as unknown as { id: string }).id : undefined;
 }
 
+const {
+  reveal: openFormFieldSelector,
+  isRevealed: isFormFieldSelectorOpen,
+  confirm,
+  cancel: closeFormFieldSelector,
+} = useConfirmDialog<never, string, never>();
+const formFieldSelector = ref<HTMLElement | null>(null);
+onClickOutside(formFieldSelector, () => closeFormFieldSelector());
```

Comment:
```suggestion
onClickOutside(formFieldSelector, closeFormFieldSelector);
```

Reply:
@anbraten this doesn't work because the expected callback function signature for the onClickOutside doesn't match cancel method signature. the callback function expects a PointerEvent